src/audio/module_adapter/module_adapter.c

```diff
@@ -1145,6 +1145,7 @@ int module_adapter_trigger(struct comp_dev *dev, int cmd)
 	return module_adapter_set_state(mod, dev, cmd);
 }
 EXPORT_SYMBOL(module_adapter_trigger);
+EXPORT_SYMBOL(module_adapter_set_state);
```

Comment:
same - next to the function. Checkpatch also complains about that https://github.com/thesofproject/sof/actions/runs/12874359088/job/35893642621?pr=9787#step:5:17

Reply:
I think you can export this into a separate patch, as it is not related to this change.